src/components/Button/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ComponentPropsWithoutRef } from "react";
+
+export interface ButtonProps extends ComponentPropsWithoutRef<"button"> {
+  children?: React.ReactNode;
+  iconStart?: JSX.Element;
+  iconEnd?: JSX.Element;
+}
+
+export function Button(props: ButtonProps) {
+  const hoverClass = "hover:bg-tertiaryAlt";
+  const activeClass = "active:bg-secondaryAlt";
+  const disabledClass =
+    "disabled:bg-[#DADADA] disabled:cursor-not-allowed disabled:text-white";
```

Comment:
Why do we need a custom color here!? We have all colors mapped in the codebase already. That's the aim of use Tailwind. Have you try that!?

Reply:
![image](https://user-images.githubusercontent.com/72019005/235935217-45095ceb-d5e5-41cc-9d33-2f400c724b96.png)
![image](https://user-images.githubusercontent.com/72019005/235935255-0b774350-5136-43d4-9fa1-696122b1c491.png)

I created a class name "infoAlt" and I used it in class

Reply:
If it's okay for you, I push again on my branch